modules/account/src/test/java/edu/stanford/spezi/module/account/manager/UserSessionManagerTest.kt

```diff
@@ -262,6 +257,25 @@ class UserSessionManagerTest {
         every { firebaseStorage.getReference(location) } returns storageReference
     }
 
+    private fun setupHasConfirmedInvitationCode() {
+        val uid = "uid"
+        val document: DocumentSnapshot = mockk {
+            every { getString("invitationCode") } returns "1234"
+        }
+        val firebaseUser: FirebaseUser = mockk {
+            every { isAnonymous } returns false
+        }
+        val documentTask: Task<DocumentSnapshot> = mockk {
```

Comment:
I have created a `mockTask` function, feel free to use that here. If you can replace other usages as well would be great, just search for ` every { isComplete } returns true`